web/src/connectors/edit-model-connector/edit-model-connector.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { FC, useCallback, useState } from 'react';
+import EditModelSettings from 'components/model/add-model-settings/add-model-settings';
+import EditModelData from 'components/model/add-model-data/add-model-data';
+import { useEditModelMutation, useModels } from 'api/hooks/models';
+import { useCategories } from 'api/hooks/categories';
+import { Model, SortingDirection } from 'api/typings';
+import { useBasements } from 'api/hooks/basements';
+import Wizard, {
+    renderWizardButtons,
+    WizardPropsStep
+} from 'shared/components/wizard/wizard/wizard';
+import { MODELS_PAGE } from 'shared/constants/general';
+import { ModelValues } from '../../components/model/model.models';
+
+import { Form, IFormApi } from '@epam/uui';
+import wizardStyles from '../../shared/components/wizard/wizard/wizard.module.scss';
+
+type EditModelConnectorProps = {
+    onModelEdited: () => void;
+    onError: (err: any) => void;
+};
+
+const initialValues: ModelValues = {
+    name: undefined,
```

Comment:
I think we it's not necessary to define default values for fields which are undefined by default